app/livechat/server/lib/Livechat.js

```diff
@@ -533,12 +533,7 @@ export const Livechat = {
 		const agentIds = [];
 		// get the agents of the department
 		if (departmentId) {
-			let agents = Livechat.getOnlineAgents(departmentId);
-
-			if (agents.count() === 0 && settings.get('Livechat_guest_pool_with_no_agents')) {
-				agents = Livechat.getAgents(departmentId);
-			}
-
+			const agents = Livechat.getAgents(departmentId);
```

Comment:
@renatobecker As far as I can see, this change will always add all agents to the inquiry. The inquiry triggers a notification which may result in an email if the agent is not online.
=> Will this change not lead to a behavior where all agents always get notified via mail of all livechats?

Reply:
Hi @mrsimpson! Actually, we were facing an issue related to `receiver` field missing in the fake `subscription` created to send notifications to online agents. I just pushed a new commit fixing this issue, as well as we'll only consider online agents to send notifications.

Thanks.